factory/processing/processComponents_test.go

```diff
@@ -655,6 +659,67 @@ func TestNewProcessComponentsFactory(t *testing.T) {
 		require.NotNil(t, pcf)
 	})
 }
+func TestProcessComponentsFactory_AddSystemVMToContainer(t *testing.T) {
+	t.Parallel()
+
+	testContainerKey := "testContainer"
+
+	t.Run("sovereign should add systemVM", func(t *testing.T) {
+		t.Parallel()
+
+		shardCoordinator := mock.NewMultiShardsCoordinatorMock(2)
```

Comment:
Perhaps use sovereign shard coordinator for consistency?

Reply:
done